packages/graphql-tool-utilities/src/polyfills.ts

```diff
@@ -1,3 +1,3 @@
 // apollo-codegen uses Object.entries and Array.flatmap
-import 'core-js/modules/es7.object.entries';
-import 'core-js/modules/es7.array.flat-map';
+import 'core-js/modules/es.object.entries';
```

Comment:
https://github.com/zloirock/core-js/blob/6226eff151809e5229bf2a114575a3b6222e221d/docs/2019-03-19-core-js-3-babel-and-a-look-into-the-future.md#packages-entry-points-and-modules-names

>In previous versions of core-js, modules with polyfills for stable ECMAScript features and ECMAScript proposals were prefixed with es6. and es7. respectively. It was a decision taken in 2014 when all the features which could be after ES6 were considered as ES7. In core-js@3 all stable ECMAScript features are prefixed with es., while ECMAScript proposals with esnext..